gettingStarted.html

```diff
@@ -44,156 +47,355 @@
+git clone https:github.com/Microsoft/vcpkg.git && ./vcpkg/bootstrap-vspkg.bat</textarea
+                            >
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
-        <div id="build" class="container-fluid">
-            <div class="row">
-                <h4 class="section-title col-md-11 offset-md-1 space">
-                    2. Build the Binary
-                </h4>
-            </div>
-            <div class="row">
-                <div class="col-md-5 offset-md-1 marg">
-                    <h5 class="device-title col-md-12">On Windows</h5>
-                    <div class="terminal col-md-12">
-                        <p>PS> &nbsp .\bootstrap-vspkg.bat</p>
-                    </div>
+            <div id="build" class="container-fluid">
+                <div class="row">
+                    <h4 class="section-title col-md-11 offset-md-1 space">
+                        Install Libraries for Your Project
+                    </h4>
                 </div>
-                <div class="col-md-5 marg">
-                    <h5 class="device-title col-md-12">On Linux or MacOS</h5>
-                    <div class="terminal col-md-12">
-                        <p>Linux:~/$ &nbsp ./bootstrap-vcpkg.sh</p>
+                <div class="row">
+                    <div class="col-md-10 offset-md-1">
+                        <div id="tab-container" class="tabs">
+                            <div class="tablist" role="tablist">
+                                <div>
+                                    <svg
+                                        id="install-copy"
+                                        onclick="copyGSCode('step2');"
+                                        viewBox="0 0 16 16"
+                                        class="bi bi-files copy-btn"
+                                        xmlns="http://www.w3.org/2000/svg"
+                                    >
+                                        <path
+                                            fill-rule="evenodd"
+                                            d="M3 2h8a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2zm0 1a1 1 0 0 0-1 1v10a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1H3z"
+                                        />
+                                        <path
+                                            d="M5 0h8a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2v-1a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1H5a1 1 0 0 0-1 1H3a2 2 0 0 1 2-2z"
+                                        />
+                                    </svg>
+                                </div>
+                            </div>
+                            <textarea
+                                id="windows-step2"
+                                class="tab-panel windows"
+                                readonly
+                            >
+.\vcpkg\vcpkg install [packages to install]</textarea
```

Comment:
Maybe we can add a link here to the list of available packages

Reply:
updated